config/crd/bases/operator.dash0.com_dash0monitorings.yaml

```
@@ -248,6 +248,13 @@ spec:
                   See https://github.com/dash0hq/dash0-operator/blob/main/helm-chart/dash0-operator/README.md#managing-dash0-check-rules-with-the-operator
                   for details. This setting is optional, it defaults to true.
                 type: boolean
+              synchronizeViews:
+                default: true
+                description: |-
+                  If enabled, the operator will watch Dash0 view resources in this namespace and create corresponding views in Dash0 via the Dash0 API.
+                  See https://github.com/dash0hq/dash0-operator/blob/main/helm-chart/dash0-operator/README.md#managing-dash0-check-rules-with-the-operator
```

issue: wrong link (points to check rules management)

```suggestion
                  See https://github.com/dash0hq/dash0-operator/blob/main/helm-chart/dash0-operator/README.md#managing-dash0-views
```